frontend/apps/reader/modules/readerkoptlistener.lua

```diff
@@ -21,10 +21,7 @@ function ReaderKoptListener:onReadSettings(config)
     normal_zoom_mode = ReaderZooming.zoom_mode_label[normal_zoom_mode] and normal_zoom_mode or ReaderZooming.DEFAULT_ZOOM_MODE
     self.normal_zoom_mode = normal_zoom_mode
     self:setZoomMode(normal_zoom_mode)
-    self.document.configurable.contrast = config:readSetting("kopt_contrast")
-                                       or G_reader_settings:readSetting("kopt_contrast")
-                                       or 1.0
-    self.ui:handleEvent(Event:new("GammaUpdate", 1/self.document.configurable.contrast))
+    self.ui:handleEvent(Event:new("GammaUpdate", self.document.configurable.contrast))
```

Comment:
Maybe move this to `ReaderView` where the rest of it is?

Reply:
We can't, the "configurable" settings are loaded in ReaderConfig that is registered in RederUI after ReaderView.

Reply:
So ReaderView handles "ReadSettings" event before ReaderConfig.